### DISCUSSION TITLE
How to Remove the Punctuation Between `title` and `titleaddon/subtitle` in DOCX Output?

### DISCUSSION THREAD

**Discussion by TomBener** [jgm/pandoc#9725] [Q&A] [answered]
I’m asking this question because I have a specific requirement for my citations. I’m working with Chinese titles where the value of `titleaddon/subtitle` is the translated title. The punctuation between the `title` and `titleaddon/subtitle` is unnecessary and I’d like to know if there’s a way to remove it.

I’ve asked on the [Zotero forum](https://forums.zotero.org/discussion/comment/462353) but it seems the issue is not related to the CSL style. In my current CSL `modern-asian-studies.csl`, there is no definition for `titleaddon` or `subtitle`. In this case, I’d like to remove the punctuation between the `title` and `titleaddon/subtitle`. E.g. removing the period `.` or colon `:` between `title` and `titleaddon/subtitle` in the following example:

![docx-screenshot](https://github.com/jgm/pandoc/assets/49151155/972af3f2-c44a-47d9-909c-4710aefe717f)

I have created a [GitHub repo](https://github.com/TomBener/custom-csl-title) to demonstrate this specific issue. If anyone has any suggestions or knows of a way to accomplish this, please let me know. I’d appreciate any help or guidance you can provide.

**Comment by jgm** [comment, marked as answer]
You're using a bibtex or biblatex bibliography, I imagine?  CSL itself doesn't have `subtitle`.  So this has to do with pandoc's conversion from bibtex/biblatex to CSL references.  The colon is hard coded; you can't change it.  But you could convert your .bib file to `csljson` using pandoc (`pandoc -f biblatex -t csljson`) and use the json file as your bibliography; you'd be able to change the punctuation in it.  Or you could convert the .bib to markdown `pandoc -f biblatex -t markdown -s` which would give you something a references section you could insert directly into your markdown file (and you could edit the title there).

**Reply by TomBener** [reply]
Thanks, converting to `csljson` is quite useful and solving the problem!

**Reply by TomBener** [reply]
While by using CSL JSON, the specific punctuations can be removed easily, this approach doesn’t have a built-in feature like biblatex’s double brackets to retain cases, which is strictly necessary in my use-case. Is it possible to use Lua filter to remove the punctuation while using biblatex? Thanks!

**Reply by jgm** [reply]
In CSL JSON, if I recall, you can do `<span class="nocase">aBcD</span>` to preserve case.

**Reply by TomBener** [reply]
It works. Huge thanks!
